scm/src/multi_run_gmtb_scm.py

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env python
 
+import sys
+PYTHON2 = sys.version_info[0] < 3
```

Comment:
Argh. This is what I did for `ccpp_prebuild.py` and it works with both Python 2 and 3. Not sure if it would work in your case:
```
import importlib
...
    # Import the host-model specific CCPP prebuild config;
    # split into path and module name for import
    configpath = os.path.abspath(os.path.dirname(configfile))
    configmodule = os.path.splitext(os.path.basename(configfile))[0]
    sys.path.append(configpath)
    ccpp_prebuild_config = importlib.import_module(configmodule)
```

Reply:
Hmm. This was just brought over from your original python 3 implementation of the run scripts. You're saying that I should switch to this? I tried this in my python 2 environment and I'm getting an ImportError exception, with or without appending the path. In practice, I had been allowing a relative path to a filename as input into this script, and if the filename starts with ../, I'm also getting an error about needing to add a package to the import_module() call.

Reply:
It seems like there a bunch of ways to do this, but none seem superior to others to me and work for python 2 and 3: https://stackoverflow.com/questions/67631/how-to-import-a-module-given-the-full-path

Reply:
I just thought that
```
import importlib
...
importlib.import_module(mymodule)
```
would be a cleaner way, but whatever works is ok.

Reply:
The following code works on my Mac python 2/3 versions (copied from an answer on StackExchange). I'm not sure why it wasn't working with your suggested code since it is very similar.

import importlib

dirname, basename = os.path.split(args.file) 
sys.path.append(dirname) 
module_name = os.path.splitext(basename)[0] 
scm_runs = importlib.import_module(module_name) 
sys.path.pop()